draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -140,9 +140,50 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 # Security Considerations
 
-TODO Security and Privacy
+## WIMSE ID
+
+The WIMSE ID is scoped within a trust domain and therefore a WIMSE ID is only unique within a trust domain. Using a WIMSE ID without taking into account the trust domain could allow one domain to issue tokens to spoof identities in another domain.  Additionally, the trust domain must be tied to an authorized trusted key through some mechanism such as a JWKS or X.509 certificate chain. The association of a trust domain and a cryptographic trust root MUST be communicated securely out of band.
+
+[TODO: Should there be a DNS name to Trust domain mapping defined?]
+
+## WIMSE ID Token and Proof of Possession
+
+The WIMSE ID token is bound to a secret cryptographic key and is always presented with a proof of possession as described in section TBD. It MUST NOT be used as a bearer token. While this helps reduce the sensitivity of the token it is still possible that a token and its proof of possession may be captured and replayed.  If this risk is mitigated then WIMSE ID tokens may be resued for a period of time with fresh proofs generated for each transaction.  The following are some mitigations for the capture and reuse of the proof of possession (POP):
+
+* Preventing Eavesdropping and Interception with TLS
+
+An attacker observing or intercepting the communication channel can view the token and its proof of possession and attempt to replay it to gain an advantage. In order to prevent this the
+token and proof of possession MUST be sent over a secure, server authenticated TLS connection unless a secure channel is provided by some other mechanisms. Host name validation according
+to section TBD MUST be performed. The WIMSE ID token itself is not usable without a proof of possession.
```

Comment:
Hmmm, not exactly. Sometimes there's a middlebox rather than end-to-end TLS, and then you have host name validation, but the "host" is the middlebox.

Reply:
Also, the Message Sig section includes replay mitigations (detection of duplicate nonces). We should mention that and have a similar mechanism in the pop-esque option.

Reply:
You still need to do host name validation.  the service will bee to know the name of the intermediary.  I'm not sure there is more to explain here. 
I added a bit about the nonces in a later section.